spec/features/user_can_sign_in_spec.rb

```diff
@@ -2,14 +2,16 @@
 
 feature 'can sign in with an existing account', :devise do
   scenario 'signs in with an account' do
-    user = FactoryGirl.create(:user)
-
-    visit root_path
-    click_link 'Sign In'
-    fill_in 'Email', with: user.email
-    fill_in 'Password', with: user.password
-    click_button 'Log in'
-
+    new_user = FactoryGirl.create(:user)
```

Comment:
You're creating a user factory object in both tests, refactor by pulling them into a single let statement